cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -256,7 +256,7 @@ if(builtin_lz4)
     URL ${lcgpackages}/lz4-${lz4_version}.tar.gz
     URL_MD5 c9610c5ce97eb431dddddf0073d919b9
     INSTALL_DIR ${CMAKE_BINARY_DIR}
-    CONFIGURE_COMMAND  /bin/sh -c "PREFIX=<INSTALL_DIR> make cmake"
+    CONFIGURE_COMMAND  /bin/sh -c "PREFIX=<INSTALL_DIR> CMAKE_PARAMS='-DCMAKE_C_COMPILER=\\\"${CMAKE_C_COMPILER}\\\" -DCMAKE_C_FLAGS=\\\"${CMAKE_C_FLAGS}\\\" -DCMAKE_OSX_SYSROOT=\\\"${CMAKE_OSX_SYSROOT}\\\"' make cmake"
```

Comment:
Why hard code the path to the shell here? It looks like the LZ4 external is using CMake, so wouldn't it be better to use `${CMAKE_COMMAND}` or similar? Otherwise the shell may pick up a different CMake.